### PR TITLE
add post about desktop release v0.104.0

### PR DESCRIPTION
**Pull request by Jikstra**

### CODE REVIEW

**Review by ralphtheninja — APPROVED** [review]

**Comment by r10s** [comment]
cool :)

@Jikstra technically: as the first paragraph is shown in the overview ([preview](https://deploy-preview-162--deltachat.netlify.com/en/blog)) it is better not to use an image there.

**Comment by r10s** [comment]
@Jikstra did a pr with the changes i suggested: https://github.com/deltachat/deltachat-pages/pull/163

**Comment by nicodh** [comment]
location-streaming

**Comment by Jikstra** [comment]
picture v0 103 0

**Comment by hpk42** [comment]
improved the blog post, as discussde with jikstra.  but integrating the screenshots is still missing. 

also to verify that no telemetry leackage happens to mapbox.com cc @nicodh

**Comment by Jikstra** [comment]
I would propose to release 0.104.0 in the next days (currently only upgrading rpgp is blocking, that pr is currently in the ci pipeline) and update this post accordingly. This means including the new and awesome fixes/improvements in the changelog (a lot better performance for example :))

File: _posts/2019-06-25-desktop.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+title: New desktop release with RPGP, location streaming and improved mac & ubuntu builds
+author: Jikstra, holga
+---
+
+<img src="../assets/blog/20190625-desktop.png" width="400" style="float: right ; width: 400" /> 
+There are new Desktop releases (0.104.0) with a new Rust-based encryption engine (RPGP),
+better Mac and Linux release packaging and several other UX improvements which 
+we detail further below.  See "Assets" on our [github releases page](https://github.com/deltachat/deltachat-desktop/releases/tag/v0.104.0) for the respective downloads. 
+
+**Import Notes about this release**:
```
Comment (ralphtheninja) [inline _posts/2019-06-25-desktop.md]: Important

File: _posts/2019-06-25-desktop.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+- **Don't forget to Synchronize Android/iOS to Desktop:** if you install Delta Chat desktop next
+  to an an already existing mobile version you need to synchronize your setups.
```
Comment (ralphtheninja) [inline _posts/2019-06-25-desktop.md]: -an

File: _posts/2019-06-25-desktop.md (hunk 1 above)
Comment (ralphtheninja) [inline _posts/2019-06-25-desktop.md]: Important

File: _posts/2019-06-25-desktop.md (hunk 2 above)
Comment (ralphtheninja) [inline _posts/2019-06-25-desktop.md]: -an

File: _posts/2019-06-25-desktop.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+  advanced settings and accept the Setup message from your Desktop app. After correctly entering 
+  the security/confirmation number (presented om mobile, typed in on desktop) you will see 
+  outgoing and incoming messages on both devices -- synchronization should be quite immediate. 
+
+- **You can use Desktop as your only Delta Chat App, no mobile required.**
+
+- **Delta Chat Desktop is well usable but does not have all UX features
+  of Android**. You are more than welcome to contribute [fixes/features on github](https://github.com/deltachat/deltachat-desktop) but note that feature discussion usually happens first 
+  on the [support forum](https://support.delta.chat). To begin with, the onboarding
+  process could be nicer for existing Android/iOS Delta Chat users could be nicer ;)
```
Comment (ralphtheninja) [inline _posts/2019-06-25-desktop.md]: two "could be nicer"

File: _posts/2019-06-25-desktop.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+  <img src="../assets/blog/20190625-desktop-location.png" width="200" style="float: left;width: 100%;margin-top: 5px;margin-bottom: 10px;" /> 
+
+- Lots of [improved Desktop packaging](download),
+  majorly pushed forward by Jikstra the last weeks: 
+  There are now **signed releases for Mac**
+  which allows to install Delta Chat Desktop without fiddling with security settings. 
+  Also openssl and all other dependencies are included in the dmg
+  install file. This means no more `brew install ...` for mac users!
+
+  There are now .deb archives for all major Ubuntu versions**
```
Comment (ralphtheninja) [inline _posts/2019-06-25-desktop.md]: Unmatched `**`

File: _posts/2019-06-25-desktop.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+
+Besides, many smaller features and fixes landed with this Desktop release: 
+
+- draft new messages are now saved on a per-chat basis 
+
+- some performance fixes which make the UI more fluid (more are coming!) 
+
+- importing and exporting keys works again 
+
+- a lot of refactoring of the backand/state handling took place,
```
Comment (ralphtheninja) [inline _posts/2019-06-25-desktop.md]: backend

File: _posts/2019-06-25-desktop.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+  to simplify future changes and maintenance. 
+
+- the desktop release now has support for simple markdown. Give it a try!
```
Comment (ralphtheninja) [inline _posts/2019-06-25-desktop.md:87]: Maybe give some examples on how it works? I.e. using backticks etc.

**Review by ralphtheninja — CHANGES_REQUESTED** [review]

**Comment by r10s** [comment]
lgtm, nice work :)

**Review by ralphtheninja — APPROVED** [review]

**Pull request merged**